modules/network-observability-configuring-FLP-sampling.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+
+// * configuring-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-config-FLP-sampling{context}"]
+
+= Configuring the Flow Collector resource
```

Comment:
I would rather remove the previous section (in the Installation part) that mentions creating the flowcollector, and keep that as "configuring". (cf my other comment in "Installing... "). The reason is that these two parts overlap .. and why not configuring it right in the first place ?

Reply:
Considering....I guess I was thinking it would be important to both demonstrate proper creation and also making updates to the YAML. They are definitely basically the same in content. Let me think about this...